src/com/shmuelzon/HomeAssistantFloorPlan/Controller.java

```diff
@@ -532,7 +560,7 @@ private List<StateIcon> generateLightsStateIcons() {
                 lightsCenter.add(getFurniture2dLocation(light));
             lightsCenter.scale(1.0 / lightsList.size());
 
-            stateIcons.add(new StateIcon(lightsList.get(0).getName(), lightsCenter, "icon", "toggle"));
+            stateIcons.add(new StateIcon(lightsList.get(0).getName(), lightsCenter, "icon", "toggle", lightsList.get(0).getDescription() != null ? lightsList.get(0).getDescription() : lightsList.get(0).getName()));
```

Comment:
I think I prefer the title to remain null and not have a tooltip with the name of the entity